framework/source/class/qx/dev/unit/TestResult.js

```diff
@@ -98,8 +98,8 @@ qx.Class.define("qx.dev.unit.TestResult",
     /**
      * Run a test function using a given test result
      *
-     * @param testResult {TestResult} The test result to use to run the test
-     * @param test {TestSuite|TestFunction} The test
+     * @param testResult {qx.dev.unit.TestResult} The test result to use to run the test
+     * @param test {qx.dev.unit.TestSuite|TestFunction} The test
```

Comment:
This is missing the full class of TestFunction. It should say,
`qx.dev.unit.TestSuite|qx.dev.unit.TestFunction`.